### DISCUSSION TITLE
Run IPED in the forensic profile

### DISCUSSION THREAD

**Discussion by Giak1234** [sepinf-inc/IPED#2076] [General]
Morning,
is there a possibility to run the tool in **forensic mode**? If yes how to run it compared to the standard command (iped.exe -d evidence.e01 -d evidencefolder)?
Thank you in advance.

**Comment by DeveloperNamedMax** [comment]
To run IPED in forensic mode you can use command "iped.exe -d image.dd -o output -profile forensic".
All the available profiles: [https://github.com/sepinf-inc/IPED/wiki/User-Manual#command-line-options](https://github.com/sepinf-inc/IPED/wiki/User-Manual#command-line-options)

**Reply by Giak1234** [reply]
Thanks
